packages/next/build/jest/jest.ts

```diff
@@ -107,10 +107,9 @@ export default function nextJest(options: { dir?: string } = {}) {
           '^.+\\.(js|jsx|ts|tsx)$': [
             require.resolve('../swc/jest-transformer'),
             {
-              styledComponents:
-                nextConfig && nextConfig.experimental.styledComponents,
-              paths,
```

Comment:
`paths` is not passed anymore?

Reply:
It's now passed through using https://github.com/vercel/next.js/pull/31376/files#diff-e8c94546832bb75bccb2c6581fdb9160237f7e6e034930f010897ea4af76b5fdR111